### PR TITLE
Metrics server

### PR DESCRIPTION
**Pull request by Hsn723**
This PR adds container builds for metrics-server which can then be used for https://github.com/cybozu-go/neco/issues/484, for instance.

### CODE REVIEW

File: metrics-server/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ARG CGO_ENABLED=0
+WORKDIR /go/src/github.com/kubernetes-incubator
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN git clone --branch v${METRICS_SERVER_VERSION} https://github.com/kubernetes-sigs/metrics-server.git && \
+    cd metrics-server && \
+    make all
+
+# Runtime stage
+FROM scratch
+COPY --from=build /go/src/github.com/kubernetes-incubator/metrics-server/_output/amd64/metrics-server /usr/local/metrics-server/bin/metrics-server
```
Comment (dulltz) [inline metrics-server/Dockerfile]: Please also copy LICENSE file

File: metrics-server/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Build stage
+FROM quay.io/cybozu/golang:1.13-bionic AS build
+
+ARG METRICS_SERVER_VERSION=0.3.6
+ARG CGO_ENABLED=0
+WORKDIR /go/src/github.com/kubernetes-incubator
```
Comment (dulltz) [inline metrics-server/Dockerfile]: In the task of `make all`, `GO111MODULE` is `on`, so you can probably build the binary at the default workspace.
This WORKDIR is not needed.

https://github.com/kubernetes-sigs/metrics-server/blob/master/Makefile#L47

File: metrics-server/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+RUN git clone --branch v${METRICS_SERVER_VERSION} https://github.com/kubernetes-sigs/metrics-server.git && \
```
Comment (dulltz) [inline metrics-server/Dockerfile]: Please download a tarball from the release page.
It is lighter than `git clone`.
https://github.com/kubernetes-sigs/metrics-server/archive/v0.3.6.tar.gz

**Comment by morimoto-cybozu** [comment]
Please add `README.md`.

File: metrics-server/Dockerfile (hunk 2 above)
Comment (Hsn723) [reply metrics-server/Dockerfile]: That's true for `master`, however its contents haven't been officially released yet. The latest release (`0.3.6`) didn't yet use go modules.

Comment (dulltz) [reply metrics-server/Dockerfile]: Sorry !!! 🙇

**Comment by morimoto-cybozu** [comment]
@Hsn723 
There seems to be no problem now.
Press "Ready for review" if you have permission.

**Comment by Hsn723** [comment]
@morimoto-cybozu Thank you for the review. Following this morning's discussion about running CI for forked repos, I'll close this PR and instead open one from a branch in the main repo since I now have access.

**Pull request closed without merging**